api/v0/proto/blob_service.proto

```diff
@@ -0,0 +1,101 @@
+syntax = "proto3";
+
+package celestia.node.v1.blob;
+
+option go_package = "github.com/celestiaorg/celestia-node/api/v1/blob";
```

Comment:
will this be generated into celestia-node? if there is still the question i would propose an API package similar to k8s for hosting the generated go code. Secondly pushing the proto to the buf registry for remote generation for other clients could be useful. happy to help with this